action.yml

```
@@ -33,5 +33,5 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - run: cd ${{github.action_path}} && yarn install && yarn build && ERC_EDITORS=${{ inputs.ERC_EDITORS }} CORE_EDITORS=${{ inputs.CORE_EDITORS }} NETWORKING_EDITORS=${{ inputs.NETWORKING_EDITORS }} INTERFACE_EDITORS=${{ inputs.INTERFACE_EDITORS }} META_EDITORS=${{ inputs.META_EDITORS }} INFORMATIONAL_EDITORS=${{ inputs.INFORMATIONAL_EDITORS }} GITHUB_TOKEN=${{ inputs.GITHUB-TOKEN }} MAINTAINERS=${{ inputs.MAINTAINERS }} NODE_ENV=production node build/src/index.js
+    - run: cd ${{github.action_path}} && yarn install && yarn build && ERC_EDITORS=${{ inputs.ERC_EDITORS }} CORE_EDITORS=${{ inputs.CORE_EDITORS }} NETWORKING_EDITORS=${{ inputs.NETWORKING_EDITORS }} INTERFACE_EDITORS=${{ inputs.INTERFACE_EDITORS }} META_EDITORS=${{ inputs.META_EDITORS }} INFORMATIONAL_EDITORS=${{ inputs.INFORMATIONAL_EDITORS }} GITHUB_TOKEN=${{ inputs.GITHUB-TOKEN }} MAINTAINERS=${{ inputs.MAINTAINERS }} PR_NUMBER=${{ inputs.PR_NUMBER }} NODE_ENV=production node build/src/index.js
```

Can we use the multiline syntax for this?

```suggestion
    - run: |
        cd ${{github.action_path}} && \
        yarn install && \
        yarn build && \
        ERC_EDITORS=${{ inputs.ERC_EDITORS }} \
        CORE_EDITORS=${{ inputs.CORE_EDITORS }} \
        NETWORKING_EDITORS=${{ inputs.NETWORKING_EDITORS }} \
        INTERFACE_EDITORS=${{ inputs.INTERFACE_EDITORS }} \
        META_EDITORS=${{ inputs.META_EDITORS }} \
        INFORMATIONAL_EDITORS=${{ inputs.INFORMATIONAL_EDITORS }} \
        GITHUB_TOKEN=${{ inputs.GITHUB-TOKEN }} \
        MAINTAINERS=${{ inputs.MAINTAINERS }} \
        PR_NUMBER=${{ inputs.PR_NUMBER }} \
        NODE_ENV=production \
        node build/src/index.js
```

---

Sounds good, but that should be done in a separate PR. This is known to work and this change would require re-testing.

---

You're changing this code, so it's probably the best time to rework it.

Is testing it painful?

---

> Is testing it painful?

Very. Takes about 30 minutes to do a full E2E test, and it has to be done manually by two people simultaneously.